releases.md

```diff
@@ -65,7 +65,7 @@ The software maintainers will not back-port fixes or features to versions outsid
 
 | Major Version | Latest Minor Version |   Status    | Initial Release | Maintenance Window Start | Maintenance Window End |
 |:--------------|:---------------------|:-----------:|:---------------:|:------------------------:|:----------------------:|
-| 1             | 1.3.x                | Maintenance |  July 12, 2021  |       May 26, 2022       |   December 31, 2023    |
+| 1             | 1.3.x                | Maintenance |  July 12, 2021  |       May 26, 2022       |   1 Year after 3.0 release date    |
```

Comment:
My OCD says let's capitalize consistently `1 Year after 3.0 Release Date` and format the table (align the `|`), but I'll shut up now.